DFC.Digital/DFC.Digital.Web.Sitefinity.CourseModule/Web.config

```diff
@@ -165,7 +165,7 @@
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="Microsoft.Spatial" publicKeyToken="31bf3856ad364e35" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-7.5.0.20627" newVersion="7.5.0.20627" />
+        <bindingRedirect oldVersion="0.0.0.0-7.2.0.0" newVersion="7.2.0.0" />
```

Comment:
This one has gone backwards, is it correct

Reply:
this matches DEV branch, and is what is tested in SIT, so if this is incorrect a new dev / sit release needs to be created?